client/eip712/data_type.go

```diff
@@ -0,0 +1,54 @@
+package eip712
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/evmos/ethermint/client/flags"
+	"github.com/evmos/ethermint/ethereum/eip712"
+)
+
+func DataTypeCommand() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
I'd also add a `Long:` field description outlining the use case for this command, what can you once you generate the data types?

Reply:
done